configuration/configuration.go

```diff
@@ -372,7 +372,7 @@ func (c *Registry) GetEnvironment() string {
 	if strings.Contains(authURL, "prod-preview.openshift.io") {
 		return "preview"
```

Comment:
Then we should change "preview" to "prod-preview" if we want to align it with UI ;)

Reply:
And yes, you are correct good sir. Fixed :)